Luminary069/LATITUDE_LONGITUDE_SUBROUTINES.agc

```diff
@@ -19,20 +19,20 @@
 
 ## NOTE: Page numbers below have not yet been updated to reflect Luminary 69.
```

Comment:
Can you remove this note now that the page numbers have been updated? Other than that, looks great! :)